### PR TITLE
Feature/48166

### PR DESCRIPTION
**Pull request by pankaj-ps**
Setup Apollo Server

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "license": "ISC",
+  "dependencies": {
+    "@babel/cli": "^7.5.5",
+    "@babel/core": "^7.5.5",
+    "@babel/node": "^7.5.5",
+    "apollo-server-express": "^2.8.1",
+    "dotenv": "^8.1.0",
+    "express": "^4.17.1",
+    "graphql": "^14.4.2",
+    "nodemon": "^1.19.1"
```
Comment (pinku4444) [inline package.json]: please install nodemon and babel in devDependencies

File: src/config/configuration.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const envVars  = process.env;
+
+
+const configuration = Object.freeze({
+
+port : envVars.PORT,
+
+}); 
+
+
```
Comment (pinku4444) [inline src/config/configuration.js]: Please remove extra space and console.log

File: src/server.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+// import * as express from 'express';
```
Comment (pinku4444) [inline src/server.js]: Please remove all commented code

**Review by pinku4444 — COMMENTED** [review]
please fix all issue.

**Review by pankaj-ps — COMMENTED** [review]
sir i have fix all issues . please review again

File: package.json (hunk 1 above)
Comment (pankaj-ps) [reply package.json]: Done .I have set nodemon and babel in devDependencies

File: src/config/configuration.js (hunk 2 above)
Comment (pankaj-ps) [reply src/config/configuration.js]: Done

File: src/server.js (hunk 3 above)
Comment (pankaj-ps) [reply src/server.js]: done . removed all commented code

**Pull request merged**